### DISCUSSION TITLE
Is it possible to update parent in maven pom with renovate?

### DISCUSSION THREAD

**Discussion by ezloj** [renovatebot/renovate#14001] [Request Help] [answered]
Fore example, I'd expect renovate to update this out of the box, but it doesn't:
```
	<parent>
		<groupId>org.springframework.boot</groupId>
		<artifactId>spring-boot-starter-parent</artifactId>
		<version>2.1.12.RELEASE</version>
	</parent>
```
I guess an option could be to use the regex manager, but this fragment is quite generic so it would interfere with maven manager?

**Comment by rarkins** [comment]
It looks like it should be the type of thing we support. Please create a minimal reproduction + bug report issue and we'll take a look.

**Reply by ezloj** [reply]
Thank you, did that here: https://github.com/renovatebot/renovate/issues/14063

**Comment by AndreVirtimo** [comment, marked as answer]
This is already working.
